packages/cli/src/commands/__tests__/test.test.js

```diff
@@ -23,31 +23,12 @@ afterEach(() => {
   jest.clearAllMocks()
 })
 
-test('Creates/resets a test db when side has api, before calling jest', async () => {
-  await handler({
-    filter: ['api'],
-  })
-
-  expect(execa.mock.results[0].value).toEqual({
-    cmd: 'yarn rw',
-    params: ['prisma db push', '--force-reset', '--accept-data-loss'],
-  })
-
-  expect(execa.mock.results[1].value.cmd).toBe('yarn jest')
-})
```

Comment:
Let me know if I'm missing something but it feels like we should keep testing for this behavior. Do we still? Or why isn't it necessary anymore if not?

Reply:
Its not testable from the CLI tests anymore :) I'm not exactly sure how to test the jest-preset!

Reply:
@jtoar resolving, feel free to unresolve